src/components/PersonLink/PersonLink.tsx

```diff
@@ -0,0 +1,28 @@
+import { Link, useSearchParams } from 'react-router-dom';
+import cn from 'classnames';
+import { FC } from 'react';
+import { Person } from '../../types';
+
+interface Props {
+  person: Person;
+}
+
+export const PersonLink: FC<Props> = ({ person }) => {
+  const [searchParams] = useSearchParams();
+  const {
+    name,
+    sex,
+    slug,
+  } = person;
+
+  return (
+    <Link
+      to={`/people/${slug}?${searchParams}`}
+      className={cn('has-text-link', {
```

Comment:
To improve the readability of the code, it is better to transfer the definition of classes to separate variables and use the ready-made result. You can see an example of use on the website of the [classnames ](https://www.npmjs.com/package/classnames)package.